packages/mobile/src/screens/chat-screen/ChatMessageListItem.tsx

```diff
@@ -45,34 +55,104 @@ const useStyles = makeStyles(({ spacing, palette, typography }) => ({
   date: {
     fontSize: typography.fontSize.xs,
     color: palette.neutralLight2
+  },
+  unreadTagContainer: {
+    marginVertical: spacing(6),
+    flexGrow: 1,
+    display: 'flex',
+    flexDirection: 'row',
+    alignItems: 'center'
+  },
+  unreadSeparator: {
+    height: 1,
+    backgroundColor: palette.neutralLight5,
+    flexGrow: 1
+  },
+  unreadTag: {
+    color: palette.white,
+    fontSize: typography.fontSize.xxs,
+    fontFamily: typography.fontByWeight.bold,
+    textTransform: 'uppercase',
+    letterSpacing: 0.5,
+    backgroundColor: palette.neutralLight5,
+    paddingHorizontal: spacing(2),
+    paddingVertical: spacing(1),
+    borderRadius: spacing(0.5)
+  },
+  tail: {
+    display: 'flex',
+    position: 'absolute',
+    zIndex: -1,
+    bottom: 47
+  },
+  tailIsAuthor: {
+    right: -spacing(3)
+  },
+  tailOtherUser: {
+    left: -spacing(3),
+    transform: [{ scaleX: -1 }]
   }
 }))
 
+const pluralize = (message: string, shouldPluralize: boolean) =>
```

Comment:
Can't wait until we just have all the i18n tools! https://react.i18next.com/misc/using-with-icu-format#plural